2016/proposals/result-aggregation-server-garg-prerit.md

```diff
@@ -0,0 +1,196 @@
+Result-aggregation server for the installation-test scripts 
+============================================================== 
+ 
+Contents 
+-------------- 
+1. **Abstract** 
+2. **Technical Details** 
+3. **Schedule of deliverables** 
+4. **Future Works** 
+5. **Open Source Development Experience** 
+6. **Academic Experience** 
+7. **Why this Project ?** 
+ 
+## Abstract 
+Software Carpentry has been teaching researchers in science, engineering, medicine, and related disciplines the computing skills they need to get more done in less time and with less pain. Several workshops are held in which students are required to setup their system by installing necessary packages. Currently software carpentry provides the installation-test scripts so that, students can check that they've successfully installed any software required by their workshop. But the results generated by the scripts are not collected anywhere. In this project a server is to be written for storing results of installation-testing scripts, and also enhancing the testing scripts by adding an option for users to collect results and send them to the server.  In this project, an API will be made so that, the results can be collected on the server, and thus can be figured out what all problems students usually face in installations. This will also help in making informed decisions about deprecating older versions of various dependencies or modernizing the lessons to catch up with current systems. 
```

Comment:
Please break down this line into multiple lines – I can't comment on specific sentences otherwise :)